workflow_miniscope/ingest.py

```diff
@@ -24,43 +24,30 @@ def ingest_sessions(session_csv_path='./user_data/sessions.csv'):
     with open(session_csv_path, newline= '') as f:
         input_sessions = list(csv.DictReader(f, delimiter=','))
 
-    # Folder structure: root / subject / session / .tif (raw)
+    # Folder structure: root / subject / session / .avi (raw)
     session_list, session_dir_list, scan_list, scanner_list = [], [], [], []
 
     for sess in input_sessions:
         sess_dir = pathlib.Path(sess['session_dir'])
 
-        # search for either ScanImage or ScanBox files (in that order)
-        for scan_pattern, scan_type, glob_func in zip(['*.tif', '*.sbx'],
-                                                      ['ScanImage', 'ScanBox'],
+        # Search for Miniscope-DAQ-V3 files (in that order)
+        for scan_pattern, scan_type, glob_func in zip(['*.avi'],
+                                                      ['Miniscope-DAQ-V3'],
                                                       [sess_dir.glob, sess_dir.rglob]):
```

Comment:
you probably want:
```
for scan_pattern, scan_type, glob_func in zip(['*.avi'],
                                                      ['Miniscope-DAQ-V3'],
                                                      [sess_dir.glob]):
```
(if this needs a recursive search, then use `sess_dir.rglob` instead)

Reply:
Thanks for the explanation.  Fixed.